app/controllers/bulk_downloads_controller.rb

```diff
@@ -1,9 +1,44 @@
-
 class BulkDownloadsController < ApplicationController
   include BulkDownloadTypesHelper
+  include BulkDownloadsHelper
+
+  before_action do
+    allowed_feature_required("bulk_downloads")
+  end
 
   # GET /bulk_downloads/types
   def types
     render json: BulkDownloadTypesHelper::BULK_DOWNLOAD_TYPES
   end
+
+  # POST /bulk_downloads
+  def create
+    # Convert sample ids to pipeline run ids.
+    begin
+      pipeline_run_ids = get_pipeline_run_ids_for_samples(bulk_download_params[:sample_ids])
```

Comment:
maybe a comment to say the access control is checked in `get_pipeline_run_ids_for_samples` :)